### DISCUSSION TITLE
챗봇 기획자료

### DISCUSSION THREAD

**Discussion by parkmina365** [InChil2/SlotTagging#16] [참고자료]
[슬롯태깅을 이용한 서브웨이 주문 챗봇](https://velog.io/@jjulme/%EC%8A%AC%EB%A1%AF%ED%83%9C%EA%B9%85%EC%9D%84-%EC%9D%B4%EC%9A%A9%ED%95%9C-%EC%84%9C%EB%B8%8C%EC%9B%A8%EC%9D%B4-%EC%A3%BC%EB%AC%B8-%EC%B1%97%EB%B4%87)

**Comment by InChil2** [comment]
[버트를 활용한 스터디룸 예약 챗봇](https://github.com/K-107/BERTsDay/blob/main/%EB%B0%9C%ED%91%9C%EC%9E%90%EB%A3%8C.pdf)

**Comment by InChil2** [comment]
[챗봇 기획 팁](https://brunch.co.kr/@hyunahpark/39)

**Comment by InChil2** [comment]
[서브웨이 챗봇 깃헙](https://github.com/k151202/BERT_sandwich_order)

**Comment by seuly1203** [comment]
[CAKD3 1기 BERT 활용한 스터디룸 예약 챗봇 (유튜브 영상)](https://youtu.be/KoPE-CMso50)
